openstf/model/confidence_interval_applicator.py

```diff
@@ -137,3 +147,24 @@ def _add_quantiles_to_forecast(
             )
 
         return forecast
+
+    def _add_quantiles_to_forecast_quantile_regression(
+        self, forecast: pd.DataFrame, quantiles: list
+    ) -> pd.DataFrame:
+        """Add quantiles to forecast.
+        Use trained quantile regression model to calculate the quantiles.
+        Args:
+            forecast (pd.DataFrame): Forecast
+            quantiles (list): List with desired quantiles
```

Comment:
voor het 'quantiles' argument zou het helpen om een voorbeeldje te zien. is dit bijv 
`[1,10,90,99]` of `[0.01, 0.1, 0.9, 0.99]` of [P_01,P10,...]` ...